### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by atmosplak** [learn-co-curriculum/intro-to-tdd-rspec-and-learn#19]
I have been having issues with various aspects, including, missing repositories, inability to clone lab, etc.

I am getting a "local build - build failures" indicator, which I believe may be false, given my checking and rechecking my submitted code.

I would like to redo this lab and am wondering if there is a way to wipe the slate clean so that I may complete the three tasks over again: Fork, Build, and Pull Request.

Thank you

**Comment by JessRudder** [comment]
Hi @atmosplak,

Sorry to hear you're having so many issues.

If you delete your repository on Github that will have the effect of wiping the slate clean.

You can then refork, clone and start over.

Make sure you are absolutely sure that's what you want to do as deleting a repo is not something that can be reversed.

**Comment by atmosplak** [comment]
Hi,

Thank you for the reply. Deleting the repository will only rid of any progress I've made on the rspec and learn section, correct?

I'll take another look at it again to see if I did anything wrong. 

Thank you for your help. 

Chris

**Comment by JessRudder** [comment]
Deleting the repository will only get rid of the work you did for the "intro-to-tdd-rspec-and-learn" lesson.

You will actually still have your local copy of the work, but it will point to a repo that doesn't exist.  If you want to get rid of that, you'll need to delete that directory on your machine.

**Comment by atmosplak** [comment]
Hello and thank you for the reply:

I deleted the repository both locally and on my machine and am still getting errors on the "local build" step when there is nothing apparently wrong with the work. I've used the learn command several times while checking the few lines of code, and have passed each time. 

I'm wondering if you all are able to offer me some assistance on either clearing my cache on your end, so that I am able to start fresh and complete the tasks over, or assist me in clearing the local build error so that I can proceed with the next lesson. 

I understand that I can proceed anyway without clearing the lesson, but I'd prefer to complete the course 100% is I progress forward. 

Thank you
